src/commands/deploy/latest.js

```diff
@@ -64,14 +64,31 @@ const addProcessEnv = async (log, env) => {
 
 const deploymentErrorMsg = `Your deployment failed. Please retry later. More: https://err.sh/now-cli/deployment-error`;
 
+const parseFinalAliases = (aliasFinal) => {
+  const last = aliasFinal.length - 1;
+
+  if (last === 0) {
+    // Only one item
+    return aliasFinal[0];
+  }
+
+  return `${aliasFinal.slice(0, last).join(', ')} and ${aliasFinal[last]}`;
+};
+
 const printDeploymentStatus = (
   output,
-  { url, readyState },
+  { url, readyState, aliasFinal },
   deployStamp,
   builds
 ) => {
   if (readyState === 'READY') {
-    output.success(`Deployment ready ${deployStamp()}`);
+    if (aliasFinal && Array.isArray(aliasFinal)) {
```

Comment:
Should we check for the length here too?

Reply:
We could, but the API will do that anyway